src/main/java/at/hannibal2/skyhanni/utils/LorenzUtils.kt

```diff
@@ -141,7 +141,7 @@ object LorenzUtils {
 
     fun getSBMonthByName(month: String): Int {
         var monthNr = 0
-        for (i in 1..12) {
```

Comment:
Kotlin does not want empty spaces before or after the .. operator. See the examples without space here: https://kotlinlang.org/docs/ranges.html